src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/indexstatemanagement/IndexStateManagementHistory.kt

```diff
@@ -177,20 +180,20 @@ class IndexStateManagementHistory(
     }
 
     @Suppress("NestedBlockDepth")
-    suspend fun addHistory(managedIndexMetaData: List<ManagedIndexMetaData>) {
+    suspend fun addManagedIndexMetaDataHistory(managedIndexMetaData: List<ManagedIndexMetaData>) {
```

Comment:
can we use Metadata 😀

Reply:
Will keep it same for now as it's following parameters convention